scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -227,7 +228,7 @@ public static BuildTargetResult PublishCoreHostPackages(BuildTargetContext c)
         {
             foreach (var file in Directory.GetFiles(Dirs.CorehostLocalPackages, "*.nupkg"))
             {
-                var hostBlob = $"{Channel}/Binaries/{CliNuGetVersion}/{Path.GetFileName(file)}";
+                var hostBlob = $"{Channel}/Binaries/{SharedFrameworkNugetVersion}/{Path.GetFileName(file)}";
```

Comment:
That sounds reasonable, and I think I misunderstood the purpose here. 

My first take was that we were trying to put artifacts in a versioned folder that matches, but I think what I'm understanding now is just that we're switching the grouping from being under the CLI version to being under the sharedframework version. That sounds fine to me.